9.5/Dockerfile.rhel7

```diff
@@ -1,4 +1,4 @@
-FROM rhel7
+FROM registry.access.redhat.com/rhel7
```

Comment:
I'm not sure whether we can do this, @hhorak/@pkubat?

Reply:
Might cause some issues as this will always try to pull the image from the registry (or expect something tagged like this locally), which likely is not what is going on in OSBS.

For postgresql this is ok, since it will get replaced by the specific base image but I would rather gave the same format in every image.